extensions/ql-vscode/src/run-queries-shared.ts

```diff
@@ -382,10 +382,10 @@ export class QueryEvaluationInfo extends QueryOutputDir {
 
 export interface QueryWithResults {
   readonly query: QueryEvaluationInfo;
+  readonly result: legacyMessages.EvaluationResult;
```

Comment:
It's keeping in line with functions where optional parameters should be at the end (I moved this so that I can compare better).
I think it doesn't hurt but might improve readability so I would like to leave it in.